engines/wintermute/base/gfx/x/modelx.cpp

```diff
@@ -104,9 +104,11 @@ bool ModelX::loadFromFile(const Common::String &filename, ModelX *parentModel) {
 	uint32 fileSize = 0;
 	byte *buffer = BaseFileManager::getEngineInstance()->getEngineInstance()->readWholeFile(filename, &fileSize);
 
-	byte *dataFormatBlock = buffer + 8;
+	byte dataFormatBlock[4];
+	Common::copy(buffer + 8, buffer + 11, dataFormatBlock);
```

Comment:
add some comments about hardcoded values here. like what it shift value is

Reply:
Added a comment